### ISSUE TITLE
NTR: Cholestanol high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8458]
New term request Cholestanol high in CSF (elthat_101120221338): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000203
- label: Elevated CSF cholestanol
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:33509302
- def: The concentration of cholestanol in the cerebrospinal fluid (CSF) is above the upper limit of normal. Cholestanol is the 5-alpha-dihydro derivative of cholesterol.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8458
- synonyms:
